test/arrayops.jl

```diff
@@ -1869,7 +1869,7 @@ end
     end
 end
 
-isdefined(:TestHelpers) || include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))
+isdefined(:TestHelpers) || eval(Main, :(include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))))
```

Comment:
No since the tests are not run in `Main` anymore but `TestHelpers` must be defined in main to avoid duplication.

This is also wrong though, the right way is https://github.com/JuliaLang/julia/pull/17435/files#diff-2da7c96ff8c6e51cfe6f8d0e1f28cf85L3